Cargo.toml

```diff
@@ -189,6 +189,7 @@ nearcore = { path = "nearcore" }
 near-crypto = { path = "core/crypto" }
 near-dyn-configs = { path = "core/dyn-configs" }
 near-epoch-manager = { path = "chain/epoch-manager" }
+near-epoch-sync-tool = { path = "tools/epoch-sync", package = "epoch-sync-tool" }
```

Comment:
Any reason you’ve chosen to specify the package name here and to rename the dependency over calling the package `near-epoch-sync-tool` in the first place?